spinnaker-monitoring/spectator_client.py

```diff
@@ -51,7 +52,7 @@ def get_source_catalog(config_dir=None):
        whose value is the dictionary of the YAML file content.
   """
   config_dir = config_dir or CONFIG_DIR
-  source_dir = os.path.join(config_dir, 'sources')
+  source_dir = SOURCES_DIR # os.path.join(config_dir, 'sources')
```

Comment:
Actually, if the default source_dir is config_dir/sources then the default SOURCES_DIR should be None and here would be SOURCES_DIR or os.path.join(config_dir, 'sources') so the default remains 'sources' relative to config_dir.

I might feel different about it in the morning. This is still ugly.